diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -642,6 +651,19 @@ namespace diff_drive_controller
         last_odom_tf_publish_time_ + publish_period_ < time + half_period &&
         tf_odom_pub_->trylock())
     {
+      // Update twist:
```

Comment:
This part is a little tricky... indeed, it could happen that the twist is updated here in this cycle and then it also gets updated in the next cycle in the odom topic publisher above :exclamation: 

I'm looking for a better solution...